lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -479,3 +486,12 @@ export async function addReleaseNotes(
 export function shouldSkipChangelogMd(repository: string): boolean {
   return repositoriesToSkipMdFetching.includes(repository);
 }
+
+function getSourceRootPath(type: ChangeLogPlatform): string {
```

Comment:
should be part of platform API as told in other PR

Reply:
Same reply https://github.com/renovatebot/renovate/pull/32689#discussion_r1855214435